assets/helpers/abTests/abtestDefinitions.js

```diff
@@ -17,4 +19,27 @@ export const tests: Tests = {
     seed: 4,
   },
 
+  annualContributionsRoundFour: {
+    type: 'AMOUNTS',
+    variants: [
+      {
+        id: 'control',
+        amountsRegions: annualAmountsControl,
```

Comment:
I feel like it should be possible to just omit this param if nothing's getting overriden?

Reply:
good point